packages/console/src/pages/CustomizeJwtDetails/MainContent/index.tsx

```diff
@@ -56,14 +58,20 @@ function MainContent<T extends LogtoJwtTokenPath>({
 
       await api.put(getApiPath(tokenType), { json: payload });
 
-      if (action === 'create') {
-        navigate(-1);
-        return;
-      }
-
       const result = await mutate();
 
       reset(formatResponseDataToFormData(tokenType, result));
+
+      /**
+       * Should `reset` (to set `isDirty` to false) before navigating back to the custom JWT listing page.
```

Comment:
Could you explain the relation between `mutateJwtCustomizers` and `reset` form?

Reply:
The form contains the properties of the current JWT customizer, in another words, its displayed by the details page. And the `mutateJwtCustomizers` updates list of all JWT customizers. Since in the flow, users will be redirected to the listing page after the creation, we'd better update the list, otherwise the list page will not show the up-to-date status.